extensions/blocks/rating-star/index.js

```diff
@@ -27,7 +28,7 @@ export const settings = {
 		_x( 'rating', 'block search term', 'jetpack' ),
 		_x( 'review', 'block search term', 'jetpack' ),
 	],
-	category: supportsCollections() ? 'formatting' : 'jetpack',
+	category: supportsCollections() ? getCategoryWithFallbacks( 'widgets', 'formatting' ) : 'jetpack',
```

Comment:
This is the same situation as with the [Newspack Blocks categories in FSE](https://github.com/Automattic/wp-calypso/pull/43670#pullrequestreview-444862833) - this is always going to fall back to `widgets` category because it's present in both, pre- & post-category-update Gutenberg versions. This makes this fallback unnecessary, but [there already was a decision to leave this as is](https://github.com/Automattic/wp-calypso/pull/43670#issuecomment-655625284), so it would make sense to leave this one as well.